### ISSUE TITLE
Support for react v16

### ISSUE THREAD

**Issue by moisesnandres** [kirillDanshin/halogenium#7]
I got this error when I updated my react version
```
TypeError: Cannot read property 'bool' of undefined
./node_modules/halogen/BounceLoader.js
node_modules/halogen/BounceLoader.js:31
  28 |  * @type {Object}
  29 |  */
  30 | propTypes: {
> 31 |     loading: React.PropTypes.bool,
  32 |     color: React.PropTypes.string,
  33 |     size: React.PropTypes.string
  34 | },
```

**Comment by kirillDanshin** [comment]
hi!

which version of halogenium you use?

**Comment by moisesnandres** [comment]
Hi @kirillDanshin 
The version is:
"halogenium": "^2.0.0-rc1"

**Comment by kirillDanshin** [comment]
thanks!

**Comment by kirillDanshin** [comment]
please checkout 2.0.0-rc2 version, this error should be fixed

**Comment by moisesnandres** [comment]
Thanks! It worked.

**Comment by kirillDanshin** [comment]
awesome! please update to just released 2.0.0 :)
